mava/networks/attention.py

```diff
@@ -36,47 +36,34 @@ def setup(self) -> None:
         self.proj = nn.Dense(self.n_embd, kernel_init=orthogonal(0.01))
 
         # causal mask to ensure that attention is only applied to the left in the input sequence
-        self.mask = jnp.tril(jnp.ones((self.n_agent + 1, self.n_agent + 1))).reshape(
-            1, 1, self.n_agent + 1, self.n_agent + 1
-        )
+        self.mask = jnp.tril(jnp.ones((self.n_agent + 1, self.n_agent + 1)))[
+            jnp.newaxis, jnp.newaxis
+        ]
```

Comment:
mega nit, not a fan of the indexing on a new line:
```suggestion
        self.mask = jnp.tril(jnp.ones((self.n_agent + 1, self.n_agent + 1)))
        self.mask = self.mask[jnp.newaxis, jnp.newaxis]
```